ctwrap/simulation.py

```diff
@@ -84,7 +100,15 @@ def run(self, name: Optional[str] ='defaults',
             config: Optional[Dict[str, Any]] = None,
             **kwargs: str) -> None:
         """
-        Run function holding configuration in dictionary.
+        Runs the simulation module's run method.
+
+        The :meth:`run` is used to call the module's run method.
+        A simple usage is
```

Comment:
be more specific. Also, there are some nouns missing ... explicitly write 'method' after 
```
:meth:`run`
```
and add 'example' after usage.

Reply:
An example is on the next line

Reply:
there is ... what I meant is that you should write 'A simple usage example is'

Reply:
Oh!